emanifest-py/src/example.py

```diff
@@ -14,18 +14,17 @@
 #   json: string                  part of multipart/mixed response if applicable
```

Comment:
Several of the methods referenced here do not reflect the updated PEP-8 naming conventions